packages/ui-progress/src/ProgressBar/styles.ts

```diff
@@ -39,7 +39,7 @@ const generateStyle = (
   componentTheme: ProgressBarTheme,
   props: ProgressBarProps
 ): ProgressBarStyle => {
-  const { valueNow, valueMax, size, color, meterColor } = props
+  const { valueNow = 0, valueMax = 100, size, color, meterColor } = props
```

Comment:
It is a bit risky to set default values for these here too, because they can get out of sync with the defaults set in defaultProps. But logically, these values will always stay like this, so probably not so risky in this case :)

Reply:
Couldn't find another sensible solution, and to moving the defaults to a separate variable and exporting it seemed a bit of an overkill. Other than that, since there are always defaults to these values, even if they get out of sync, these values in styles.js will be overwritten every time